[ADD] subscriptions: add ecommerce configuration (#4270)
odoo/documentation: content/applications/sales/subscriptions/ecommerce.rst

=======================================
Use subscriptions in the eCommerce shop
=======================================

Subscription products can be sold in the Odoo *eCommerce* shop just like regular sales products.

However, by default, the eCommerce product page will only display the shortest recurrence period
listed in the :guilabel:`Time-based pricing` tab of the product form. For example, if a
subscription product has Monthly and Yearly recurrence periods configured, then only the Monthly
price will appear on the eCommerce page for that product.

To add more recurrence periods to the eCommerce product page, create the different recurrence

> periods as *product variants*.

Reviewer: It might be good to link to the documentation for creating subscription products and product variants since some of the configuration options below don't appear by default.

=======================================
Use subscriptions in the eCommerce shop
=======================================

Subscription products can be sold in the Odoo *eCommerce* shop just like regular sales products.

However, by default, the eCommerce product page only displays the shortest recurrence period
listed in the :guilabel:`Time-based pricing` tab of the product form. For example, if a

> subscription product has Monthly and Yearly recurrence periods configured, then only the Monthly

Reviewer 2: ```suggestion
subscription product has *monthly* and *yearly* recurrence periods configured, then only the monthly
```

> However, by default, the eCommerce product page only displays the shortest recurrence period
> listed in the :guilabel:`Time-based pricing` tab of the product form. For example, if a
> subscription product has *monthly* and *yearly* recurrence periods configured, then only the monthly
> price appears on the eCommerce page for that product by default.

Reviewer 3: 100th char breaks.
I shouldn't see these by the time they hit final review 😉 
cc: @samueljlieber 
```suggestion
However, by default, the eCommerce product page only displays the shortest recurrence period listed
in the :guilabel:`Time-based pricing` tab of the product form. For example, if a subscription
product has *monthly* and *yearly* recurrence periods configured, then only the monthly price
appears on the eCommerce page for that product by default.
```

To add more recurrence periods to the eCommerce product page, create a *product variant* for each
recurrence period.

.. seealso::

>    - :doc:`products`

Reviewer 3: Think we can improve the title slightly and do global path here, like the others.
cc: @samueljlieber 
```suggestion
   - :doc:`Products </applications/sales/sales/products_prices/products>`
```

>    - :doc:`Product variants</applications/sales/sales/products_prices/products/variants>`

Reviewer 2: ```suggestion
   - :doc:`Product variants </applications/sales/sales/products_prices/products/variants>`

```

> Configuration
> =============

Reviewer 3: [Be specific with headings](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#titles-and-headings)
```suggestion
Create recurrence periods as product variants
=============================================
```

> To create the recurrence periods as product variants, go to :menuselection:`Subscriptions -->
> Subscriptions --> Products` and select a product. In the :guilabel:`Attributes & Variants` tab,
> click :guilabel:`Add a line`.

Reviewer 3: With the heading change, this paragraph requires an update
```suggestion
First, set the new recurence period(s) on the product form by navigating to
:menuselection:`Subscriptions --> Subscriptions --> Products` and select a product. In the
:guilabel:`Attributes & Variants` tab, click :guilabel:`Add a line`.
```

Create an :guilabel:`Attribute` called `Billing Period` (or something similar) by typing in the
name and clicking :guilabel:`Create`. This attribute name appears as the option heading on the
product page of the eCommerce shop.

Next, create :guilabel:`Values` that correspond to the recurrence periods that are configured in
the :guilabel:`Time-based pricing` tab of the product form. Type in the name of the recurrence
period, then click :guilabel:`Create`. These value names appear as selectable options on the
product page of the eCommerce shop.

.. image:: ecommerce/recurrence-period-attributes-variants.png
   :align: center
   :alt: Recurrence periods configured as product variants in the "Attributes & Variants" tab of
         the product form.

> Click on the cloud icon at the top of the page to save manually. After saving, a :guilabel:`Product

Reviewer 2: ```suggestion
Click on the :guilabel:`☁️ (cloud)` icon at the top of the page to save manually. After saving, a :guilabel:`Product
```
